rtsp.go

```diff
@@ -27,11 +31,16 @@ import (
 	"go.viam.com/rdk/resource"
 )
 
+/*
+#include <libavcodec/avcodec.h>
+*/
+import "C"
+
 var family = resource.ModelNamespace("viam-labs").WithFamily("viamrtsp")
-var ModelH264 = family.WithModel("rtsp-h264")
+var Model = family.WithModel("rtsp")
 
 func init() {
-	resource.RegisterComponent(camera.API, ModelH264, resource.Registration[camera.Camera, *rtsp.Config]{
+	resource.RegisterComponent(camera.API, Model, resource.Registration[camera.Camera, *rtsp.Config]{
 		Constructor: func(ctx context.Context, _ resource.Dependencies, conf resource.Config, logger logging.Logger) (camera.Camera, error) {
```

Comment:
We can pass the NewRTSPCamera directly into the constructor without wrapping it in the function by deferring getting the resource.Named and the config converter logic into that function. We can also unexport the creator function, nothing is using it in the resource manager or this package I think.

Reply:
- Un-exported init functions.
- Made `newRTSPCamera` the direct constructor.